frontend/packages/console-shared/src/components/formik-fields/field-types.ts

```diff
@@ -4,6 +4,8 @@ export interface FieldProps {
   name: string;
   label?: string;
   helpText?: React.ReactNode;
+  helperTextInvalid?: React.ReactNode;
+  validated?: boolean;
```

Comment:
Maybe this should be on `InputFieldProps`?
Not a big fan of tri-state booleans. Maybe use PatternFly's `ValidatedOptions`, or at least add a comment.

Reply:
Updated